.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
      - main
```

Same for other workflows

Reply:
Oops yes, sorry I copy-pasted from one of my other repos :sweat_smile: